src/services/mqtt_client/mqtt_listener.py

```diff
@@ -85,13 +90,18 @@ def get_value_topic_prefix(self) -> str:
             self.config.topic
         ))
 
+    def get_schedule_value_topic_prefix(self) -> str:
+        return self.__make_topic((self.get_listener_topic_prefix(), 'schedules'))
+
     @exception_handler
     def _on_message(self, client, userdata, message: MQTTMessage):
         logger.debug(f'Listener Topic: {message.topic}, Message: {message.payload}')
         with self.__app_context():
             if not message.payload:
                 return
-            if self.get_listener_topic_prefix() in message.topic:
+            if self.get_schedule_value_topic_prefix() in message.topic:
+                self.__dispatch_schedule_value_event(message)
```

Comment:
As in others, replace with the function `check_and_clear_schedule_value_topic` and that function should do:

- parse `name` or `uuid` from the topic and match it with our existing `ScheduleModel`:
  - If it got matched: then `dispatch` values if it's `cloud` and do nothing for `local`.
  - Else (on the unmatched case): we clear out the value as in other examples. So on the next subscription, we won't have those schedule values which do not really exist.